docs/further.md

```diff
@@ -160,6 +160,15 @@ set-resources:
         cpus_per_task: 40
 ```
 
+### Additional Command Line Flags
+
+This plugin defines additional command line flags. As always the can be used on the command line or in a profile.
+
+| Flag        | Meaning  |
+|-------------|----------|
+| `--slurm_init_seconds_before_status_checks`| will modify the default time (40 seconds) before the initial status check - usefull for development purposes|
```

Comment:
Just expanding a bit on the reasoning behind the default and where shorter wait times might be desirable.
```suggestion
| `--slurm_init_seconds_before_status_checks`| modify time before initial job status check; the default of 40 seconds avoids load on querying slurm databases, but shorter wait times are for example useful during workflow development |
```

Reply:
thank you